source/basic.tex

```diff
@@ -1752,6 +1752,32 @@
 If that lookup finds nothing, it undergoes unqualified name lookup;
 in each case, only names
 that denote types or templates whose specializations are types are considered.
+\begin{example}
+\begin{codeblock}
+struct T1 { struct U { int i; }; };
+struct T2 { };
+struct U1 {};
+struct U2 {};
+
+struct B {
+  using T = T1;
+  using U = U1;
+  operator U1 T1::*();
+  operator U1 T2::*();
+  operator U2 T1::*();
+  operator U2 T2::*();
+};
+
+template<class X, class T>
+int g() {
+  using U = U2;
+  X().operator U T::*();        // \#1, searches for \tcode{T} in the scope of \tcode{X} first
+  X().operator U decltype(T())::*();
```

Comment:
Presumably this should be marked `#2`?  Also, `g` doesn't return anything; maybe replace `int x=…;` with `void f() {…}`.

Reply:
Fixed.